src/main/java/org/elasticsearch/script/expression/ExpressionScriptEngineService.java

```diff
@@ -60,6 +61,13 @@
     protected static final String GET_MINUTES_METHOD      = "getMinutes";
     protected static final String GET_SECONDS_METHOD      = "getSeconds";
 
+    protected static final String MINIMUM_METHOD          = "min";
+    protected static final String MAXIMUM_METHOD          = "max";
+    protected static final String AVERAGE_METHOD          = "avg";
+    protected static final String MEDIAN_METHOD           = "med";
```

Comment:
I think "median" is a better function name here? Limiting to 3 letters doesn't seem to buy us anything.